docs/interfaces/cli-ug.rst

```diff
@@ -35,7 +35,7 @@ and upgrade.
  Installation
 **************
 
-The CLI is distributed as a Python wheel package and requires Python >= 3.7. We recommend setting up
+The CLI is distributed as a Python wheel package and requires Python >= 3.8. We recommend setting up
 a `virtualenv <https://virtualenv.pypa.io/en/latest/>`__ and using the ``pip`` utility to install
 ``determined`` into the environment:
 
```

Comment:
The Command Line Interface (CLI) is distributed in the form of a Python wheel package and requires Python version 3.8 or later. 

(source: [Microsoft Manual of Style](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/later))

Reply:
Done.